### PR TITLE
SITL: fix parse error on start

### PR DESCRIPTION
**Pull request by julianoes**
~Two things that caught my eye. See commits.~

This fixes the error:
```etc/init.d-posix/rcS: 39: [: Illegal number:```

### CODE REVIEW

File: src/modules/logger/logged_topics.cpp [hunk 1]
```diff
@@ -467,7 +467,7 @@ void LoggedTopics::add_mission_topic(const char *name, uint16_t interval_ms)
 bool LoggedTopics::add_topic(const orb_metadata *topic, uint16_t interval_ms, uint8_t instance, bool optional)
 {
 	if (_subscriptions.count >= MAX_TOPICS_NUM) {
-		PX4_WARN("Too many subscriptions, failed to add: %s %" PRIu8, topic->o_name, instance);
+		PX4_DEBUG("Too many subscriptions, failed to add: %s %" PRIu8, topic->o_name, instance);
```
Comment (dagar) [inline src/modules/logger/logged_topics.cpp:470]: We should probably fix the root cause here instead. I'll take a look.

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: We should, yes. But I don't know it...

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: Any idea what to do @bkueng?

Comment (bkueng) [reply src/modules/logger/logged_topics.cpp:470]: You can either increase `MAX_TOPICS_NUM` which requires more stack, and/or remove some of the topics.

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: Or ignore the warning? :D

Comment (bkueng) [reply src/modules/logger/logged_topics.cpp:470]: That's hardly a good solution. It means it will silently not log certain things.

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: Ok, let me say this again: all I care about is not to see a warning, it's embarrassing if doing the default SITL startup we get spammed with warnings that aren't relevant.

Comment (dagar) [reply src/modules/logger/logged_topics.cpp:470]: I'll trim down multi-ekf across all mags by default for now, at least until it factors in the configured priority.

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: Ping.

Comment (dagar) [reply src/modules/logger/logged_topics.cpp:470]: https://github.com/PX4/PX4-Autopilot/pull/21614

Comment (julianoes) [reply src/modules/logger/logged_topics.cpp:470]: Thanks, let me test this and rebase.

**Comment by julianoes** [comment]
Force pushed. Can probably go in now.

**Comment by julianoes** [comment]
Seems to work for me, merging.

**Pull request merged**